bot/utils/time.py

```diff
@@ -161,71 +267,58 @@ def relativedelta_to_timedelta(delta: relativedelta) -> datetime.timedelta:
     return utcnow + delta - utcnow
 
 
-def time_since(past_datetime: datetime.datetime) -> str:
-    """Takes a datetime and returns a discord timestamp that describes how long ago that datetime was."""
-    return discord_timestamp(past_datetime, TimestampFormats.RELATIVE)
-
-
-def parse_rfc1123(stamp: str) -> datetime.datetime:
-    """Parse RFC1123 time string into datetime."""
-    return datetime.datetime.strptime(stamp, RFC1123_FORMAT).replace(tzinfo=datetime.timezone.utc)
+def format_relative(timestamp: Timestamp) -> str:
+    """
+    Format `timestamp` as a relative Discord timestamp.
 
+    A relative timestamp describes how much time has elapsed since `timestamp` or how much time
+    remains until `timestamp` is reached.
 
-def format_infraction(timestamp: str) -> str:
-    """Format an infraction timestamp to a discord timestamp."""
-    return discord_timestamp(dateutil.parser.isoparse(timestamp))
+    `timestamp` can be any type supported by the single-arg `arrow.get()`, except for a `tzinfo`.
```

Comment:
I think here (and in other docstrings) the param typehint alone should be enough without further explaining it in the docstring; if necessary the comment above the type alias explains where exactly the types come from

Reply:
I don't know... I think it doesn't hurt to keep these docstrings just to be safe on the clarity of what it is.

Reply:
At least they'd have enough information to look up the arrow docs.